### PR TITLE
Monthly newsletter: June

### PR DESCRIPTION
**Pull request by awvwgk**
It's again time to prepare our monthly newsletter. Going forward we want to start earlier to give everybody enough time to contribute and also be able to publish on time.

- [x] add stdlib updates
- [x] add fpm updates
- [x] add fortran-lang.org updates
- [x] add monthly call link
- [ ] add LLVM / classic flang updates (@AlexisPerry)
- [x] add LFortran updates (@certik)
- [x] update contributor data (@LKedward)

As usual, please add your name to the authors list if you add, edit, or review content.

---

@zmoon I used your script from #261 to prepare this post. Worked great to collect the relevant links, I'll add some feedback soon.

### CODE REVIEW

File: _posts/2021-06-01-Fortran-Newsletter-June-2021.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+### LFortran
+
+
+## Events
+
+* We had our 12th Fortran Monthly call on May 20.
+You can watch the recording below:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/06hVFA8ApG4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
```
Comment (milancurcic) [inline _posts/2021-06-01-Fortran-Newsletter-June-2021.md:128]:
```suggestion

* Google Summer of Code program has announced the allocation of students to each project.
  Fortran-lang received six studens (one through [NumFOCUS](https://numfocus.org/)) who will work across three subprojects: stdlib, fpm, and LFortran.
  Congratulations and welcome to students
  [Aman Godara](https://github.com/aman-godara),
  [Rohit Goswami](https://github.com/haozeke),
  [Jakub Jelínek](https://github.com/kubajj),
  [Chetan Karwa](https://github.com/chetankarwa),
  [Thirumalai Shaktivel](https://gitlab.com/Thirumalai-Shaktivel), and
  [Gagandeep Singh](https://github.com/czgdp1807).
  Read the full post [here](https://fortran-lang.org/newsletter/2021/05/18/Welcome-GSoC-students/).
  
```

File: _posts/2021-06-01-Fortran-Newsletter-June-2021.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: "Fortran newsletter: June 2021"
+category: newsletter
+author: Sebastian Ehlert
```
Comment (milancurcic) [inline _posts/2021-06-01-Fortran-Newsletter-June-2021.md:5]:
```suggestion
author: Sebastian Ehlert, Milan Curcic
```

**Review by milancurcic — APPROVED** [review]

File: _posts/2021-06-01-Fortran-Newsletter-June-2021.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title: "Fortran newsletter: June 2021"
+category: newsletter
+author: Sebastian Ehlert, Milan Curcic
```
Comment (LKedward) [inline _posts/2021-06-01-Fortran-Newsletter-June-2021.md:5]:
```suggestion
author: Sebastian Ehlert, Milan Curcic, Laurence Kedward
```

**Review by LKedward — APPROVED** [review]

**Comment by awvwgk** [comment]
@certik @AlexisPerry Do you have updates for the compiler sections we can include?

@LKedward @milancurcic @certik Could any of you check the settings tab of this repo whether we can enable GitHub's new automerge feature for this repository? I would like to test creating an automerge for the 1st June.

**Comment by LKedward** [comment]
Sure, but just to check, if we enable auto-merge will this PR not merge immediately because it already has two reviews and checks passed?

**Comment by awvwgk** [comment]
That's what I wanted to check, maybe we can set a delay for merging. Also, this repo has no requirements for merges and no CI that has to pass, everyone with push access can merge right away.

**Comment by LKedward** [comment]
I've enabled it for you to try, but I'm not sure what you have in mind is possible currently

**Comment by certik** [comment]
Yes, I have updates that I will send later today.

**Review by certik — APPROVED** [review]
Ok, I am done!

**Pull request merged**